integrations/gitlab.js

```diff
@@ -8,49 +8,47 @@ const { gitIssues } = require('../utils/api/gitlab')
  */
 async function listIssues(authData) {
   const { gitlabToken, projectId } = authData
-  sdk.log('')
-  ux.spinner.start('Retrieving GitLab issues')
+  await ux.print('')
+  await ux.spinner.start('Retrieving GitLab issues')
   const issues = await gitIssues(gitlabToken, projectId)
-  ux.spinner.stop('Done!')
+  await ux.spinner.stop('Done!')
 
   if (!issues.length) {
-    sdk.log(ux.colors.magenta('\nNo issues found!'))
+    await ux.print(ux.colors.magenta('\nNo issues found!'))
```

Comment:
Can you update the messages to include some emoji's to improve the UX.

Emoji Guide here: https://www.notion.so/ctoai/Emoji-Icon-Guide-8a89bb0e48714387b356ec8457e22cde

For an example of what to aim for try out the demo Customize Op or Deploy Op. You can run them on any of the aws-demo teams, let me know if you need access

Reply:
I dont think I'm on the slack channel for those, so can you send me an invite when you can?

Reply:
Invited! Thanks for giving it a facelift!